.build/docker/run-tests.sh

```diff
@@ -202,14 +201,18 @@ docker_command="source \${CASSANDRA_DIR}/.build/docker/_set_java.sh ${java_versi
 docker_id=$(docker run --name ${container_name} ${docker_flags} ${docker_mounts} ${docker_volume_opt} ${image_name} sleep 4h)
 
 echo "Running container ${container_name} ${docker_id}"
+#sleep 600
 
 docker exec --user root ${container_name} bash -c "\${CASSANDRA_DIR}/.build/docker/_create_user.sh cassandra $(id -u) $(id -g)" | tee -a ${logfile}
 docker exec --user root ${container_name} update-alternatives --set python /usr/bin/python${python_version} | tee -a ${logfile}
 
+docker cp /home/jenkins/agent/workspace/k8s-e2e/build ${container_name}:/home/cassandra/cassandra
+
 # capture logs and pid for container
 docker exec --user cassandra ${container_name} bash -c "${docker_command}" | tee -a ${logfile}
 status=$?
-
+docker cp ${container_name}:/home/cassandra/cassandra/build/test/output /home/jenkins/agent/workspace/k8s-e2e/build/test
+docker exec --user cassandra ${container_name} bash -c "mkdir -p /home/cassandra/cassandra/result/1/${target} && cp -r /home/cassandra/cassandra/build/test/output /home/cassandra/cassandra/result/1/${target}"
```

Comment:
we can't be doing k8s/jenkins stuff in this script.

Reply:
For tests, the `${DIST_DIR}` (inside the docker container) is originally defined here: https://github.com/thelastpickle/cassandra/blob/infracloud/cassandra-5.0/.build/docker/ubuntu2004_test.docker#L20

and mounted here: https://github.com/thelastpickle/cassandra/blob/infracloud/cassandra-5.0/.build/docker/run-tests.sh#L84